fvm/src/gas/price_list.rs

```diff
@@ -416,9 +536,104 @@ impl PriceList {
     pub fn on_verify_consensus_fault(&self) -> GasCharge<'static> {
         GasCharge::new("OnVerifyConsensusFault", self.verify_consensus_fault, 0)
     }
+
+    /// Returns the gas required for the specified exec_units.
+    #[inline]
+    pub fn on_consume_exec_units(&self, exec_units: u64) -> GasCharge<'static> {
+        GasCharge::new(
+            "OnConsumeExecUnits",
+            self.gas_per_exec_unit.saturating_mul(exec_units as i64),
```

Comment:
to be safe, we should either take `exec_units` as an i64, or do a saturating conversion. I mean, this is almost _certainly_ safe... but we don't want to wrap around.